src/components/WeatherWidget/WeatherWidget.tsx

```diff
@@ -124,9 +126,16 @@ export default function WeatherWidget({ location }: Props) {
                         alt='Weather condition'
                       />
                     </Tooltip>
-                    {`Temperature: ${roundTemperature(
-                      temp,
-                    )}°C Humidity: ${humidity}%`}
+                    {`Temperature: ${roundTemperature(temp)}${(() => {
```

Comment:
Extract the function that pretty-prints temperature units out of markup similar to `roundUnits`